core/include/jiminy/core/robot/BasicTransmissions.h

```diff
@@ -0,0 +1,60 @@
+#ifndef JIMINY_BASIC_TRANSMISSIONS_H
+#define JIMINY_BASIC_TRANSMISSIONS_H
+
+#include "jiminy/core/robot/AbstractTransmission.h"
+
+
+namespace jiminy
+{
+    class SimpleTransmission : public AbstractTransmissionBase
+    {
+    public:
+        ///////////////////////////////////////////////////////////////////////////////////////////////
+        /// \brief      Dictionary gathering the configuration options shared between transmissions
+        ///////////////////////////////////////////////////////////////////////////////////////////////
+        virtual configHolder_t getDefaultTransmissionOptions(void) override
+        {
+            // Add extra options or update default values
+            configHolder_t config = AbstractTransmissionBase::getDefaultTransmissionOptions();
+            config["mechanicalReduction"] = 0.0;
+
+            return config;
+        };
+
+        struct transmissionOptions_t : public abstractTransmissionOptions_t
+        {
+            float64_t const mechanicalReduction;    ///< Gear reduction ratio motor to joint
+
+            transmissionOptions_t(configHolder_t const & options) :
+            abstractTransmissionOptions_t(options),
+            mechanicalReduction(boost::get<float64_t>(options.at("mechanicalReduction")))
+            {
+                // Empty.
+            }
+        };
+
+    public:
+        SimpleTransmission(std::string const & name);
+        virtual ~SimpleTransmission(void) = default;
+
```

Comment:
overload initialize method, in order to limit the type of joints that are supported and only 1-to-1 mapping for simplicity:
```
// SimpleTransmission only support for 1-to-1 mapping, for linear and rotary joints
if (jointTypes_[i] != joint_t::LINEAR && jointTypes_[i] != joint_t::ROTARY && jointTypes_[i] != joint_t::ROTARY_UNBOUNDED)
```